bottlerocket-settings-models/settings-extensions/kubernetes/src/lib.rs

```diff
@@ -96,6 +96,8 @@ pub struct KubernetesSettingsV1 {
     node_ip: IpAddr,
     pod_infra_container_image: SingleLineString,
     hostname_override: ValidLinuxHostname,
+    ids_per_pod: KubernetesIdsPerPodValue,
```

Comment:
We use i64 directly in the shape.

https://github.com/bottlerocket-os/bottlerocket-settings-sdk/blob/93069665154c90f8ea3a01f4a78972f20dea7d8e/bottlerocket-settings-models/settings-extensions/kubernetes/src/lib.rs#L74


For consistency, let's just do:

```suggestion
    ids_per_pod: i64,
```

And we can drop the custom model type `KubernetesIdsPerPodValue`

Reply:
This API has a restriction on the value so the type seems right:

> IDsPerPod is the mapping length of UIDs and GIDs. The length must be a multiple of 65536, and must be less than 1<<32.

Reply:
Yep, exactly -- that's why I had to create the custom type. Let me know if there's a better way to enforce those constraints

Reply:
Synced on this - I think this is more like a preference of whether we want to let kubelet handle the validation or we do the validation in advance. 

Either way makes sense so I will not block this.